runtime/onert/core/src/exec/IPermuteFunction.h

```diff
@@ -247,11 +288,27 @@ class IPermuteFunction : public IFunction
       const auto copy_axis = loop_shape.rank() - 1;
       const auto copy_len = loop_shape.dim(copy_axis) * sizeof(T);
       loop_shape.dim(copy_axis) = 1;
-      ShapeLoop(loop_shape, [&](const onert::ir::Coordinates &coords) {
-        // Copy src tensor's data to dst_buffer with calculated offset of dst tensor
-        memcpy(dst_buffer + dst->calcOffset(coords), src->buffer() + src->calcOffset(coords),
-               copy_len);
-      });
+
+      if (src->is_dynamic())
+      {
+        assert(dst->is_dynamic());
+        CopyDynamic(src, dst, dst_buffer, loop_shape, copy_len);
+      }
+      else
+      {
+        // TODO Comment out the assertion below
+        // assert(!dst->is_dynamic() || dst is output of graph);
```

Comment:
maybe better to remove `TODO` ?

Reply:
I mean to enable commented out code again. I don't know the opposite of "comment out".

Reply:
maybe "uncomment"

Reply:
I was confused about the meaning of `comment out` and tried to find the meaning:
https://english.stackexchange.com/questions/33483/when-i-say-comment-out-does-it-mean-to-uncomment-something-or-comment-it/33502
Interestingly `comment out` means changing code to comment. :-) So, current code was already commented out. 
As @chunseoklee mentioned, `uncomment` is right for this code. :open_mouth:

Reply:
Thanks. I will fix it.